package.json

```diff
@@ -62,7 +62,7 @@
     "markdown-it-synapse-table": "^1.0.6",
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.32",
-    "plotly.js-basic-dist": "^1.54.7",
+    "plotly.js-basic-dist-min": "^1.58.4",
```

Comment:
this ends up having no effect on portals build size, but I left it in since I think we should probably be importing the min dist